homework-submission/week1/src/components/listBody/index.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import ListItem from '../items';
+
+function ListBody({ items }) {

```

Comment:
Not saying it is wrong, but I was a bit lost in all the components you made. I think this ListBody component for example isn't really necessary. It good to create many reusable components, but too many components can also make your code harder to read for other developers 😉

Reply:
I understand what you  mean. Now we have just started to learn React I am always thinking about how many components should I have? :)
Since we have learned about Components, I tried to use them. But I thought that What if I want to change items dynamically? I think It is easy to do it with ListBody Component. So I created it. 👍